src/pattern-library/components/PlaygroundApp.js

```diff
@@ -32,14 +32,15 @@ export default function PlaygroundApp({
 }) {
   const routes = getRoutes();
   const allRoutes = routes.concat(extraRoutes);
-  const [activeRoute, navigate] = useRoute(baseURL, allRoutes);
-  const content = activeRoute ? (
-    <activeRoute.component />
-  ) : (
-    <Library.Page title=":( Sorry">
-      <h1 className="text-2xl">Page not found</h1>
-    </Library.Page>
-  );
+  const [activeRoute] = useRoute(baseURL, allRoutes);
+  const content =
+    activeRoute && activeRoute.component ? (
```

Comment:
Some entries in `allRoutes` don't have components, as they are navigation placeholders.

Reply:
The data structure for `route`s is an accretion over time. What they really are could probably more accurately be described as "pages", which may or may not have a route associated with them. Might make sense to re-assess how they're structured at some point but it's not a burning thing (IMHO) at this time.